modules/identity-provider-google-about.adoc

```diff
@@ -0,0 +1,15 @@
+// Module included in the following assemblies:
+//
+// * authentication/identity_providers/configuring-google-identity-provider.adoc
+
+[id="identity-provider-google-about_{context}"]
+= About Google authentication
+
+Using Google as an identity provider allows any Google user to authenticate to your server. You can limit authentication to members of a specific hosted domain with the `hostedDomain` configuration attribute.
+
+[NOTE]
+====
+Using Google as an identity provider requires users to get a token using `<master>/oauth/token/request` to use with command-line tools.
+====
+
+// TODO: Can we replace <master> with <control_plane_node>?
```

Comment:
@stlaz @ibihim We're trying to move away from using "master" in the docs. Would it be accurate to replace the `<master>` replaceable text on line 12 with `<control_plane_node>`?